src/compiler/diagnosticMessages.json

```diff
@@ -1023,7 +1023,7 @@
         "category": "Error",
         "code": 1354
     },
-    "A 'const' assertion can only be applied to a string, number, boolean, array, or object literal.": {
+    "A 'const' assertion can only be applied to a string, number, boolean, array, literal enum, or object literal.": {

```

Comment:
Maybe `"enum member"` instead of `"literal enum"`? I don't think most people would think of `A.b` as a "literal" of some kind, much as we often treat it like one.

Reply:
What about `"member of literal enum"`?  `"enum member"` is also weird and cannot explain that why some enum cannot be used

BTW: Honestly, I don't know the kind of enum before.

Reply:
Paging @DanielRosenwasser for error message wording

Reply:
```
'const' assertions can only be applied to references to enum members, or string, number, boolean, array, or object literals.
```